src/plugins/intel_gpu/src/kernel_selector/cl_kernels/ctc_greedy_decoder_ref.cl

```diff
@@ -7,8 +7,12 @@
 KERNEL(ctc_greedy_decoder_ref)(const __global INPUT0_TYPE* probabilities
                               ,const __global INPUT1_TYPE* sequence_indicators
                                     ,__global OUTPUT_TYPE* output_sequences
-#ifdef SECOND_OUTPUT_EXIST
+#ifdef LEGACY_MULTIPLE_OUTPUTS
+#ifdef NEW_MULTIPLE_OUTPUTS
```

Comment:
why NEW_MULTIPLE_OUTPUTS is checked within LEGACY_MULTIPLE_OUTPUTS..? NEW should be defined with LEGACY too? Sounds not intuitive.

Reply:
Fixed code to use them independently. Thanks for your comments.